paper/paper.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 'JankFlow: A Flexible Python Library for Organic Workflows'
+tags:
+  - molecular simulation
+  - materials science
+  - molecular dynamics
+  - polymers
+  - HOOMD-blue
+authors:
+  - name: Chris Jones
+    orcid: 0000-0002-6196-5274
+    equal-contrib: true
+    affiliation: 1
+  - name: Marjan Albooyeh
+    orcid: 0009-0001-9565-3076
+    equal-contrib: true
+    affiliation: 1
+  - name: Rainier Barrett
+    orcid: 0000-0002-5728-9074
+    equal-contrib: false
+    affiliation: 1
+  - name: Eric Jankowski
+    orcid: 0000-0002-3267-1410
+    corresponding: true
+    affiliation: 1
+affiliations:
+ - name: Boise State University, Boise, ID, USA
+   index: 1
+date: 01 January 2001
+bibliography: paper.bib
+---
+# Summary
+`JankFlow` is a package for reproducibly performing complex HOOMD-blue [@hoomd_2019]
```

Comment:
I still think the first sentence should be a little more descriptive than using `complex HOOMD-blue simulation workflows`. 
Probably need to clarify what do we mean by "complex workflow"?

Reply:
I agree, whether we do that in the summary or the statement of need, I still think we need to make the point of how certain workflows are harder than others. Welding, surface wetting is harder than calculating heat capacity, and Tg. At least from a process design perspective.